components/org.wso2.carbon.status.dashboard.web/src/gadgets/chart-components/NumberChart.jsx

```diff
@@ -11,109 +11,102 @@
  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
  * See the License for the specific language governing permissions and
  * limitations under the License.
- *
  */
 
 import React from 'react';
 import PropTypes from 'prop-types';
-import {VictoryLabel } from  'victory';
+import { VictoryLabel } from 'victory';
 
-export default class NumberCharts extends React.Component{
-    constructor(props){
+export default class NumberCharts extends React.Component {
+    constructor(props) {
         super(props);
-        this.state={
+        this.state = {
             height:props.config.height || 450,
             width:props.config.width || 800,
             value:null,
             prevValue:null
         };
     }
 
-
-    componentDidMount(){
+    componentDidMount() {
         this._handleData(this.props);
     }
 
-    componentWillReceiveProps(nextProps){
+    componentWillReceiveProps(nextProps) {
         this._handleData(nextProps);
     }
 
-
     /**
      * handles data received by the props
      * @param props
      * @private
      */
-    _handleData(props){
-        console.info('awa');
-        let {config,data,metadata} = props;
-        let {prevValue,value} = this.state;
-        let xIndex=metadata.names.indexOf(config.x);
-
-
-        if(data.length>0){
-            prevValue=value;
-            value=data[data.length-1][xIndex];
+    _handleData(props) {
+        let { config,data,metadata } = props;
+        let { prevValue,value } = this.state;
+        let xIndex = metadata.names.indexOf(config.x);
+
+        if (data.length > 0) {
+            prevValue = value;
+            value = data[data.length - 1][xIndex];
         }
 
-        this.setState({value,prevValue});
-
+        this.setState({ value,prevValue });
     }
 
-    render(){
-        let {config} = this.props;
-        let {width,height,prevValue,value}=this.state;
-
+    render() {
+        let { config } = this.props;
+        let { width,height,prevValue,value } = this.state;
 
-        return(
+        return (
             <svg height={'100%'} width={'100%'} viewBox={`0 0 ${width} ${height}`}>
                 <VictoryLabel
-                    textAnchor="middle" verticalAnchor="middle"
-                    x={width / 2} y={height/5}
-
+                    textAnchor="middle"
+                    verticalAnchor="middle"
+                    x={width / 2}
+                    y={height / 5}
                     text={config.title}
-                    style={{fontSize: 30}}
+                    style={{ fontSize: 30 }}
                 />
                 <VictoryLabel
-                    textAnchor="middle" verticalAnchor="middle"
-                    x={width / 2} y={height / 2}
-
-                    text={(value===null ? value : value.toFixed(3))}
-                    style={{fontSize: 45}}
+                    textAnchor="middle"
```

Comment:
formatting issues.. spaces before and after the operator

Reply:
Check other places as well

Reply:
Fixed in bbb066d61f625e5e28b09b91ec2e5a2dc09ed919

Reply:
Move to separate repo.